src/app/sequence-editor/_sequence-editor-theme.scss

```diff
@@ -85,4 +85,8 @@
   .selected-list-item {
     background-color: mat.get-theme-color($theme, secondary-container);
   }
+  #search-bar:focus {
+    outline: 1px solid #ddf0c8;
+    background-color: #dcefc7;
```

Comment:
I don't think the color is distinctive enough. Also, use a color from the theme, not a hard coded color.

<img width="346" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/c1125772-f31c-4b2b-8619-0e6df2583e44">